experiment_scheduler/master/master.py

```diff
@@ -154,29 +156,21 @@ def get_task_status(self, request, context):
         return response
 
     @start_end_logger
-    @io_logger
     def get_task_log(self, request, context):
         """
+        [Todo] Need to get info of task_manager and log_file_path through DB
         get log certain task
         :param request:
         :param context:
-        :return: log path
+        :return: log_file which is byte format and sent by streaming
         """
-        if request.task_id in dict(self.queued_tasks).keys():
-            response = self._wrap_by_task_status(
-                request.task_id, TaskStatus.Status.NOTSTART
-            )
-        elif request.task_id in dict(self.running_tasks).keys():
-            response = self.process_monitor.get_task_log(
-                self.running_tasks[request.task_id]["task_manager"], request.task_id
-            )
-            if response.status == TaskStatus.Status.KILLED:
-                del self.running_tasks[request.task_id]
+        task_manager_address = self.task_managers_address[0]
+        task_logfile_path = os.getcwd()
+        if task_logfile_path == "":
+            yield TaskLogFile(log_file=None, error_message=bytes(f"Check task id", "utf-8"))
```

Comment:
It returns response with no log_file but error_message when task_log_path no exist.